docs/quickstarts/sso-quickstart.md

```diff
@@ -0,0 +1,128 @@
+
+![A screenshot of the prompts and answers for the Yeoman generator](../images/yo-office-sso-excel.png)
+
+After you complete the wizard, the generator creates the project and installs supporting Node components.
+
+[!include[Yeoman generator next steps](../includes/yo-office-next-steps.md)]
+
+## Explore the project
+
+The add-in project that you've created with the Yeoman generator contains code for an SSO-enabled task pane add-in.
+
+- The **./ENV** file in the root directory of the project defines constants that are used by the add-in project.
+- The **./manifest.xml** file in the root directory of the project defines the settings and capabilities of the add-in.
+- The **./src/helpers/documentHelper.js** file uses the Office JavaScript library to add the data from Microsoft Graph to the Office document.
+- The **./src/helpers/fallbackauthdialog.html** file is the UI-less page that loads the fallback authentication method's JavaScript.
+- The **./src/helpers/fallbackauthdialog.js** file contains the fallback authentication method's JavaScript that signs on the user with msal.js.
+- The **./src/helpers/fallbackauthhelper.js** file contains the task pane JavaScript that invokes the fallback authentication method in scenarios when SSO authentication is not supported.
+- The **./src/helpers/ssoauthhelper.js** file contains the JavaScript call to the SSO API, `getAccessToken`, receives the bootstrap token, initiates the swap of the bootstrap token for an access token to Microsoft Graph, and calls to Microsoft Graph for the data.
+- The **./src/taskpane/taskpane.html** file contains the HTML markup for the task pane.
+- The **./src/taskpane/taskpane.css** file contains the CSS that's applied to content in the task pane.
+- The **./src/taskpane/taskpane.js** file contains the Office JavaScript API code that facilitates interaction between the task pane and the Office host application.
+
+## Configure SSO
+
+At this point, your add-in project has been created and contains the code that's necessary to facilitate the SSO process. Next, complete the following steps to configure SSO for your add-in.
+
+1. Navigate to the root folder of the project.
+
+    ```command&nbsp;line
+    cd "My SSO Office Add-in"
+    ```
+
+2. Run the following command to configure SSO for the add-in.
+
+    ```command&nbsp;line
+    npm run configure-sso
+    ```
+
+    > [!WARNING]
+    > This command will fail if your tenant is configured to require two-factor authentication. In this scenario, you'll need to manually complete the Azure app registration and SSO configuration steps manually, as described in the [Create a Node.js Office Add-in that uses single sign-on](../develop/create-sso-office-add-ins-nodejs.md) tutorial.
+
+3. A web browser window will open and prompt you to sign in to Azure. Sign in to Azure using your Office 365 administrator credentials. These credentials will be used to register a new application in Azure and configure the settings required by SSO.
```

Comment:
Should we add a note saying that if they don't sign in as an administrator, they will be prompted to sign in again? I don't know if we want to encourage people to go down the limited path, but I feel like some people will hit it anyways (even by accident). I also think it might be a good place to call out to folks that they might be signing into a non-administrator account and if they continue they will have a limited experience

Reply:
@nokafor does this note verbiage accurately describe what you want to convey here?

[!NOTE]
If you sign in to Azure using non-administrator credentials during this step, the `configure-sso` script won't be able to provide administrator consent for the add-in to users within your organization. SSO will therefore not be available to users of the add-in and they'll be prompted to sign-in.

Reply:
Yes. However, I was more trying to call out that the browser will open twice for the developer to sign in before it can register the add-in. Does that make sense? For example, when I sign in with my microsoft email the browser will open twice and I have to log into my microsoft account twice in order for it to properly register the add-in

Reply:
Interesting. If I sign-in (to Azure) with my @microsoft.com credentials (not a domain admin), I see the behavior you've described -- browser opens a second time to prompt me to sign in again. But If I sign in with my @hotmail.com credentials (also not a domain admin), I'm only prompted to sign in to Azure once (and everything works after that). So it seems that whether you get prompted to login again may depend not only on whether your account is a domain admin, but also on the type of account?

I'd suggest we not document this, for now, and instead just monitor for any feedback about folks getting confused at this step (i.e., due to the second login prompt). Then we can add clarifying documentation here later, if warranted. Sound good?

Reply:
@nokafor if you're okay with this approach, can you please approve this PR?